Documentation/teaching/lectures/address-space.rst

```diff
@@ -492,8 +492,8 @@ address.
             |                     |                 |            |                       |                   |         /----------------------\
             |                     |                 |            |                       |                   |         |                      |
```

Comment:
This commit doesn't seem to have the subject as we discussed:

`Documentation: teaching: address-space: Fix incorrect label in linux paging figure`

you only added the correct subject to github PR, but that will be lost once we merge the patch.

We want to actual commit to have the message.

You can just do:

```
$ git commit --amend
# fix the commit
$ git push -f  # this will update the existing PR (no need to create a new one).

```